job/client/src/main/java/alluxio/job/plan/replicate/DefaultReplicationHandler.java

```diff
@@ -9,7 +9,7 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.job.replicate;
+package alluxio.job.plan.replicate;
```

Comment:
Not something introduced by this PR - does this class belong here or as part of the `core/master` module?

Reply:
Probably? But not something I can answer.